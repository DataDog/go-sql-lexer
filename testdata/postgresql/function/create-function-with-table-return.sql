CREATE OR REPLACE FUNCTION get_users() RETURNS TABLE(user_id integer, user_name text) AS $func$
BEGIN
  RETURN QUERY SELECT id, name FROM users;
END;
$func$ LANGUAGE plpgsql;
