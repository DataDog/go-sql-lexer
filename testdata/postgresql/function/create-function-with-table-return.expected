[
    {
        "expected": "CREATE OR REPLACE FUNCTION get_users ( ) RETURNS TABLE ( user_id integer, user_name text ) AS $func$BEGIN RETURN QUERY SELECT id, name FROM users; END$func$ LANGUAGE plpgsql"
    },
    {
        "obfuscator_config": {
            "dollar_quoted_func": false
        },
        "expected": "CREATE OR REPLACE FUNCTION get_users ( ) RETURNS TABLE ( user_id integer, user_name text ) AS ? LANGUAGE plpgsql"
    }
]